AardvarkSample/AardvarkSample/AppDelegate.swift

```diff
@@ -29,8 +29,9 @@ class SampleAppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         
         // This line is all you'll need to get started.
-        bugReporter = Aardvark.addDefaultBugReportingGestureWithEmailBugReporter(withRecipient: "fake-email@aardvarkbugreporting.src")
-        
+        let bugReporter = Aardvark.addDefaultBugReportingGestureWithEmailBugReporter(withRecipient: "fake-email@aardvarkbugreporting.src")
+        self.bugReporter = bugReporter
```

Comment:
If you combine these two lines back together, Line 45 will no longer be ambiguous. It's currently unclear whether you're meaning to set the delegate on the property or the local version (though if it has reference semantics, I guess it doesn't matter — I'm not familiar enough with the Aardvark code to know)